Core/Object Arts/Dolphin/Base/OS.NTLibrary.cls

```diff
@@ -7,7 +7,7 @@ External.PermanentLibrary subclass: #'OS.NTLibrary'
 	classInstanceVariableNames: ''
 	classConstants: {}!
 OS.NTLibrary guid: (Core.GUID fromString: '{8293e458-7d81-4aba-97a5-c1001c4b2153}')!
-OS.NTLibrary comment: 'NtDllLibrary is the <ExternalLibrary> class to represent the dynamic link library, ''ntdll.dll''.It was generated generated from type information in the ''Win32 API'' library. It contains methods for each of the functions defined by the corresponding module in that type library.
+OS.NTLibrary comment: 'NtDllLibrary is the <ExternalLibrary> class to represent the dynamic link library, ''ntdll.dll''. It was generated from type information in the ''Win32 API'' library. It contains methods for each of the functions defined by the corresponding module in that type library.
```

Comment:
The method `TKindModuleAnalyzer>>printClassCommentPreambleOn:` should be corrected to prevent this typo reoccurring.

Reply:
I fixed this in my tree commit 47776a358 (Some comment consistency cleanups) almost three weeks ago. I should have made this into a pull request too. Sorry about that.